src/main/java/de/tum/in/www1/artemis/service/dataexport/DataExportQuizExerciseCreationService.java

```diff
@@ -105,6 +115,14 @@ private String createExportForMultipleChoiceAnswerQuestion(MultipleChoiceSubmitt
         return stringBuilder.toString();
     }
 
+    /**
+     * Adds an explanation to the answer option if no result should be included in the export
+     * The explanation contains information if the answer option was selected or not or it is invalid
```

Comment:
```suggestion
     * The explanation contains information if the answer option was selected or not or if it is invalid.
```

At least my grammar checker get's triggered by this :)